DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -1,111 +1,189 @@
-﻿namespace DigitalLearningSolutions.Data.Tests.Services

```

Comment:
This seems to be changed into unix line endings on your branch 👀

Reply:
You are correct - and I see what happened. My earlier comment mentioned RegistrationDataService not being changed - I must have completely forgotten that it wasn't RegistrationDataService that had a conflict, it was RegistrationServiceTests that had a conflict. As such, it did indeed seem to be the merge conflict resolution that caused the LF troubles. I'll quickly clean that up to fix it.